react-visualizer/src/reqJson/klis21-system.json

```
@@ -0,0 +1,313 @@
+                                        "leaf": {
+                                            "regexp_number": "^GE508[1-3]2$",
+                                            "regexp_name": ""
+                                        }
+                                    },
+                                    "知識情報システム実習A": {
+                                        "max_certificated_credit_num": 1,
+                                        "min_certificated_credit_num": 1,
+                                        "leaf": {
+                                            "regexp_number": "^GE70113$",
+                                            "regexp_name": ""
+                                        }
+                                    },
+                                    "知識情報システム実習B": {
+                                        "max_certificated_credit_num": 1,
+                                        "min_certificated_credit_num": 1,
+                                        "leaf": {
+                                            "regexp_number": "^GE70123$",
+                                            "regexp_name": ""
+                                        }
+                                    }
+                                }
+                            },
+                            "選択": {
+                                "max_certificated_credit_num": 44,
+                                "min_certificated_credit_num": 24,
+                                "leaf": {
+                                    "ＧＥ７(知識情報システム実習を除く)": {
+                                        "max_certificated_credit_num": 28,
+                                        "min_certificated_credit_num": 16,
+                                        "leaf": {
+                                            "regexp_number": "^GE7(?!.*01[1-2]3).*$",
+                                            "regexp_name": ""
+                                        }
+                                    },
+                                    "ＧＡ４／ＧＥ４／ＧＥ６／ＧＥ８": {
+                                        "max_certificated_credit_num": 36,
+                                        "min_certificated_credit_num": 8,
+                                        "leaf": {
+                                            "regexp_number": "^(GA4)|(GE(4|6|8))$",
+                                            "regexp_name": ""
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "専門基礎": {
+                "max_certificated_credit_num": 53,
+                "min_certificated_credit_num": 29,
+                "leaf": {
+                    "": {
+                        "max_certificated_credit_num": 53,
+                        "min_certificated_credit_num": 29,
+                        "leaf": {
```

```suggestion
            "専門基礎": {
                "max_certificated_credit_num": 71,
                "min_certificated_credit_num": 51,
                "leaf": {
                    "": {
                        "max_certificated_credit_num": 71,
                        "min_certificated_credit_num": 51,
                        "leaf": {
```

専門基礎としては71(=19+32)単位まで認められることが可能性に見えます
また、最低51(=19+32)単位が必要に見えます

---

最大71(=19+*52*)ってことですよね？
ありがとうございます🙇